README.md

```diff
@@ -9,6 +9,11 @@ Basics
 * New roles should be initiated in line with the skeleton directory, which has standard boilerplate
   code for a Galaxy-compatible Ansible role and some enforcement around these standards
 
+Meta Best Practices
+===================
+
+A rationale should be included for best practices, with a reference if applicable. It is really helpful to know not only how to do certain things, but why to do them in this way. It will also help with further revisions of the standards as some items may become obsolete or no longer applicable.  If the reason is not included, there is a risk of keeping items that are no longer applicable, or alternatively blindly removing items that should be kept. It also has great educational value for understanding how things actually work (or how they don’t).
+
```

Comment:
I would move it above, just below the introductory sentence. It is even more basic than the "Basics".